edge/cmd/edgecore/app/server.go

```diff
@@ -76,6 +85,23 @@ offering HTTP client capabilities to components of cloud to reach HTTP servers r
 	return cmd
 }
 
+// runningEnvironmentCheck check the environment before edgecore start
+// if Check failed,  return errors
+func runningEnvironmentCheck() error {
+
+	//if kubelet is running, return error
+	if err := osexec.Command("pgrep", "^kubelet$").Run(); err == nil {
+		return errors.New("Error: kubelet is running, kubelet should not running on edge node ")
```

Comment:
"Error: kubelet is running, kubelet should not running on edge node " ->"Kubelet should not running on edge node "